deploy/confs/gs/app.conf

```diff
@@ -33,8 +33,9 @@ web-server {
 
 # Configuration of the observe engine
 observe-engine {
-  # host for the odb
-  odb = "wss://lucuma-odb-development.herokuapp.com/ws"
+  # urls for the odb
+  odb-http = "https://lucuma-postgres-odb-production.herokuapp.com/odb" # Production ODB Server
+  odb-ws = "wss://lucuma-postgres-odb-production.herokuapp.com/ws" # Production ODB Server
```

Comment:
Are we still using the ws client?

Reply:
Yes, on the UI. This UI configuration comes from here.